PyRoute/TradeCodes.py

```diff
@@ -26,6 +26,41 @@ class TradeCodes(object):
     pcolor = {'As': '#8E9397', 'De': '#d17533', 'Fl': '#e37dff', 'He': '#ff6f0c', 'Ic': '#A5F2F3',
```

Comment:
For the `research` station code please add  `'RsO': '\u03A9'` so we can properly display it on the map(s)

Reply:
Added